docs/src/pages/homepage.module.css

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
Perhaps this file should be named `index.module.css`

Reply:
agree

Reply:
Okay, will rename.